Bulldozer.CSV/Maps/Benevolence.cs

```diff
@@ -228,7 +228,45 @@ private int LoadBenevolenceRequest( CSVInstance csvData )
                     if ( string.IsNullOrWhiteSpace( benevolenceRequestRequestedById ) )
                     {
                         // Handle Address
-                        var requestAddress = new LocationService( lookupContext ).Get( benevolenceRequestAddress.Left( 100 ), benevolenceRequestAddress2.Left( 100 ), benevolenceRequestCity, benevolenceRequestState, benevolenceRequestZip, benevolenceRequestCountry, verifyLocation: false );
+
+                        // Default country to US if not provided
+                        if ( string.IsNullOrWhiteSpace( benevolenceRequestCountry ) )
+                        {
+                            benevolenceRequestCountry = "US";
+                        }
+                        else
+                        {
+                            // Remove whitespace from country to search for and normalize various renderings of United States to US
+                            var famCountryString = new string( benevolenceRequestCountry.ToCharArray().Where( c => !Char.IsWhiteSpace( c ) ).ToArray() );
+                            if ( famCountryString.ToLower() == "unitedstates" )
+                            {
+                                benevolenceRequestCountry = "US";
+                            }
+                        }
+                        Location requestAddress = null;
+
+                        if ( !string.IsNullOrWhiteSpace( benevolenceRequestAddress ) && !string.IsNullOrWhiteSpace( benevolenceRequestCity ) && !string.IsNullOrWhiteSpace( benevolenceRequestState ) )
```

Comment:
Will this cause international addresses with no state/province to be unsupported?